ProteoformSuiteInternal/Lollipop.cs

```diff
@@ -472,7 +480,6 @@ public void clear_aggregation()
         public bool neucode_heavy_lysine = false;
         public int max_ptms = 4;
         public int decoy_databases = 1;
-        public string decoy_database_name_prefix = "DecoyDatabase_";
```

Comment:
This is definitely minor; whatever you want to do is okay. That said, you could use a field like this for consistent naming of the decoy communities.